Ductus.FluentDocker/Model/Containers/ContainerConfig.cs

```diff
@@ -1,11 +1,18 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 
 namespace Ductus.FluentDocker.Model.Containers
 {
   public sealed class ContainerConfig
   {
     public string Hostname { get; set; }
```

Comment:
I think if we camelcase domainname we should do the same on hostname

Reply:
Wikipedia writes [Hostname][1] as a single word but [Domain name][2] as two separate words. Also, both Rider and the macOS spellcheckers agree with Wikipedia. Therefore, I think that `Hostname` and `DomainName` is the correct combination even if they don't look symmetric.

[1]: https://en.wikipedia.org/wiki/Hostname
[2]: https://en.wikipedia.org/wiki/Domain_name